Frends.RabbitMQ.Publish/Frends.RabbitMQ.Publish/Frends.RabbitMQ.Publish.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
 	<TargetFrameworks>net6.0</TargetFrameworks>
-	<Version>1.1.0</Version>
+	<Version>1.1.1</Version>
```

Comment:
You have added a new parameter, so you should increase minor version by one instead of build version.